Gemfile.lock

```diff
@@ -266,125 +268,127 @@ GEM
       minitest (>= 5.1)
       tzinfo (~> 2.0)
       zeitwerk (~> 2.3)
-    addressable (2.8.4)
+    addressable (2.8.6)
       public_suffix (>= 2.0.2, < 6.0)
-    airbrake (11.0.1)
+    airbrake (11.0.3)
       airbrake-ruby (~> 5.1)
-    airbrake-ruby (5.2.0)
+    airbrake-ruby (5.2.1)
       rbtree3 (~> 0.5)
     ansible (0.2.2)
     ar_multi_threaded_transactional_tests (0.5.0)
       activerecord (>= 4.2.0, < 6.2.0)
-    ast (2.4.1)
+    ast (2.4.2)
     attr_encrypted (3.1.0)
       encryptor (~> 3.0.0)
-    audited (5.4.2)
+    audited (5.4.3)
       activerecord (>= 5.0, < 7.2)
       request_store (~> 1.2)
-    autoprefixer-rails (9.4.8)
+    autoprefixer-rails (9.8.6.5)
       execjs
-    awesome_print (1.6.1)
-    aws-eventstream (1.2.0)
-    aws-partitions (1.329.0)
-    aws-sdk-core (3.100.0)
-      aws-eventstream (~> 1, >= 1.0.2)
-      aws-partitions (~> 1, >= 1.239.0)
+    awesome_print (1.9.2)
+    aws-eventstream (1.3.0)
+    aws-partitions (1.887.0)
+    aws-sdk-core (3.191.0)
+      aws-eventstream (~> 1, >= 1.3.0)
+      aws-partitions (~> 1, >= 1.651.0)
+      aws-sigv4 (~> 1.8)
+      jmespath (~> 1, >= 1.6.1)
+    aws-sdk-ecr (1.69.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
-    aws-sdk-ecr (1.3.0)
-      aws-sdk-core (~> 3)
-      aws-sigv4 (~> 1.0)
-    aws-sdk-kms (1.23.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-kms (1.77.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.45.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-s3 (1.143.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sdk-kms (~> 1)
-      aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.6.1)
+      aws-sigv4 (~> 1.8)
+    aws-sigv4 (1.8.0)
       aws-eventstream (~> 1, >= 1.0.2)
-    base64 (0.1.1)
+    base64 (0.2.0)
+    bigdecimal (3.1.6)
     binding_of_caller (0.8.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.4.5)
-      msgpack (~> 1.0)
+    bootsnap (1.18.3)
+      msgpack (~> 1.2)
     bootstrap-sass (3.4.1)
       autoprefixer-rails (>= 5.2.1)
       sassc (>= 2.0.0)
     bootstrap3-datetimepicker-rails (4.17.47)
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
+    connection_pool (2.4.1)
```

Comment:
### [connection_pool](https://github.com/mperham/connection_pool)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.2.1 → 2.4.1

[_[change-log](https://github.com/mperham/connection_pool/blob/main/Changes.md), ~~source-code~~, [gem-diff](https://my.diffend.io/gems/connection_pool/2.2.1/2.4.1)_]





<details>
<summary>Commits</summary>

A change of **85** commits. See the full changes on [the compare page](https://github.com/mperham/connection_pool/compare/v2.2.1...v2.4.1).

These are the first 10 commits:
- (1f6109d) [update rubies](https://github.com/mperham/connection_pool/commit/1f6109d821f9e3bca7345c30857d42b41818c3a3)
- (7317ac8) [Add 2.4.1 to travis rubies](https://github.com/mperham/connection_pool/commit/7317ac897fa9b98a1789176c2789757ae5d2e721)
- (7413fbf) [optimize with (15% gain)](https://github.com/mperham/connection_pool/commit/7413fbf13f1475573fc4aa469c8e01246f0e2812)
- (8ac6a0c) [Make size/available stats available for a connection pool.](https://github.com/mperham/connection_pool/commit/8ac6a0c78f216066dd04870816d4a16d38d4d37c)
- (cd6f0e0) [Use separate methods for stats.](https://github.com/mperham/connection_pool/commit/cd6f0e0bfba923a994ef6e807b295fe27a5377f7)
- (b4ce270) [Massage method naming, wrapper support, #97](https://github.com/mperham/connection_pool/commit/b4ce270ef1e30c5660b0ae28320809efd7cad160)
- (6e86113) [add changelog, #97](https://github.com/mperham/connection_pool/commit/6e86113cd85f7aba75176717a2aec0ec8f9d1c28)
- (6dea066) [bump](https://github.com/mperham/connection_pool/commit/6dea06663c80cf50cd6b5a410376943d328b4f1e)
- (08fb434) [[CI] Test against Ruby 2.5](https://github.com/mperham/connection_pool/commit/08fb4343b28da3005a27a3e5887f438a5ff6861f)
- (b00f063) [cleanup](https://github.com/mperham/connection_pool/commit/b00f06350d30430284d754a55d25b902bae097bf)


</details>